src/main/java/org/testng/internal/ant/ReporterConfig.java

```diff
@@ -0,0 +1,48 @@
+package org.testng.internal.ant;
+
+import org.testng.collections.Lists;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Used with the &lt;reporter&gt; sub-element of the Ant task
+ *
+ * <p>NOTE: this class needs to be public. It's used by TestNG Ant task
+ */
+public class ReporterConfig {
```

Comment:
@juherr - Too many classes with the same name. Kind of confuses. Do you think we can rename this to something else ?

Reply:
Agree. `AntReporterConfig`?

Reply:
Yeah that would be better.

Reply:
Done